testgrid/updater/state.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+message Column {
+  string build = 1;  // Unique instance of the job, typically BUILD_NUMBER from prow or guid
+  reserved 2; // Useless
```

Comment:
Consider 'reserved' at top of message like in config.proto?

Reply:
Ack

As long as the messages are organized by number, I actually prefer having them this way. Is there a readability guideline published somewhere that suggests otherwise?

Reply:
Nope, none found on quick look. Seems fine to me ^^